helm/block-producer/values.yaml

```diff
@@ -75,3 +75,16 @@ healthcheck:
 
 nodeSelector:
   preemptible: true
+
+watchman:
+  enable: true
```

Comment:
change to false + note the following vars for watching core dumps should be expressed in terraform and not as default values